Source/Other/SSHTunnel/SPSSHTunnel.m

```diff
@@ -847,6 +847,7 @@ - (void)workerGetPasswordForQuery:(NSString *)theQuery
 	
 	[sshPasswordDialog setFrame:windowFrameRect display:NO];
 	[parentWindow beginSheet:sshPasswordDialog completionHandler:nil];
```

Comment:
Shouldn't this actually run the modal itself?

Reply:
The modal is not able to accept enough UI interactions when it is invoked with NSModalPanelRunLoopMode during ssh reconnection.